synapse/tools/modrole.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import yaml
+import asyncio
+import argparse
+
+import synapse.exc as s_exc
+import synapse.telepath as s_telepath
+
+import synapse.lib.output as s_output
+
+descr = '''
+Add or modify a role in a Synapse service.
```

Comment:
Is this just duplicating the cellauth tool?

Reply:
Duplicating? No.  Implementing a usable / intuitive tool that does most of the same things that the cellauth tool can do, but without being so confusing, yes :)